core/vm/dynamic_params.go

```diff
@@ -33,6 +33,14 @@ const (
 
 var getAddressForFuncABI, _ = abi.JSON(strings.NewReader(getAddressForABI))
 
+func GetRegisteredAddress(evm *EVM, registryId common.Hash) (common.Address, error) {
+	addr, err := GetRegisteredAddressWithEvm(registryId, evm)
+	if err != nil {
+		return common.ZeroAddress, nil
```

Comment:
Is this `nil` intentional?  The current behavior (from `GetRegisteredAddressWithEvm`) is to return the error (which is then checked and returned if non-nil in `(c *transfer) Run()`.

Reply:
Looks like this is modified in a later commit, so nevermind.